src/services/shims.ts

```diff
@@ -346,6 +347,14 @@ namespace ts {
             return scriptSnapshot && new ScriptSnapshotShimAdapter(scriptSnapshot);
         }
 
+        public getScriptKind(fileName: string): ScriptKind {
+            try {
+                return this.shimHost.getScriptKind(fileName);
+            } catch (e) {

```

Comment:
Removed try-catch block in favor of a check to see if the shimmed host supports getScriptKind